images/footerLinkedin.svg

```diff
@@ -0,0 +1,2 @@
+<svg xmlns="http://www.w3.org/2000/svg" class="MuiSvgIcon-root" focusable="false" viewBox="0 0 24 24" aria-hidden="true">
```

Comment:
let's name / store this as e.g. `images/footer/linkedin.svg`

Reply:
how do I do that?